src/analyses/symbLocks.ml

```diff
@@ -145,23 +145,6 @@ struct
     | _ ->
       ust
 
-  let may_race (ctx1,ac1) (ctx,ac2) =
```

Comment:
Could you explain why we removed this?

Reply:
This was removed by @vesalvojdani early on: https://github.com/goblint/analyzer/commit/1ce6e7889419f8e96acc551537752ea485b23ebc#diff-5a9240ccb6206e2ebe2d89a00ae618eea08946c5569e44e0a63a2c9456784ce8L146-L162. I think it was a leftover from some older-style race checking that wasn't actually used any more.